openedx/features/partners/give2asia/views.py

```diff
@@ -44,6 +45,13 @@ def dashboard(request, partner_slug):
                                                                        'slug': partner_slug})
 
 
+def performance_dashboard(request, partner):
+    if request.user.has_perm('can_access_g2a_performance'):
+        return render_to_response('features/partners/g2a/performance_dashboard.html',
+                                  {'slug': partner.slug, 'performance_url': partner.performance_url})
```

Comment:
more this dictionary to above line and use `dict()` to form an object

Reply:
I am wondering what would be the benefit?